test/files/glob-source.spec.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+/* eslint-env mocha */
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const chaiAsPromised = require('chai-as-promised')
+const globSource = require('../../src/files/glob-source')
+const all = require('async-iterator-all')
+const path = require('path')
+const isNode = require('is-node')
```

Comment:
can you please use https://github.com/ipfs/js-ipfs-utils/blob/master/src/env.js#L15 instead ?

Reply:
Done